languages/csharp/exercises/concept/operator-overloading/.docs/introduction.md

```diff
@@ -0,0 +1,23 @@
+The principal arithmetic and comparison operators can be adapted for use by your own classes and structs. This is known as _operator overloading_.
+
+Syntax Example:
```

Comment:
Instead of this, maybe describe that operators are static methods with the special `operator` modifier?

Reply:
I see your point but I cannot find any authorities other than ES that follow this interpretation.  The docs say "Use the operator keyword to declare an operator.".  I imagine that the rules on methods require that they are named with standard C# identifiers which obviously operators are not. 

I always enjoy ploughing an unorthodox furrow so let me know if you want me to break new ground by making this change.

Reply:
> I always enjoy ploughing an unorthodox furrow so let me know if you still want me to break new ground by making this change.

I think I explained things badly :) Instead of listing the exact syntax (which the current version does), I was looking more for something descriptive that makes the student understand that operators are like regular methods, but with a slightly different syntax. I think this might help the student understand what they can do inside the definition of a custom operator, as they can re-use everything they know about methods.

Reply:
Done